agnpy/tests/test_absorption_mu_s.py

```diff
@@ -0,0 +1,116 @@
+# tests on the utils module
```

Comment:
can we move this in the `test_absorption.py`? Just copying the entire `TestAbsorptionMuS` class over there

Reply:
To have a single test unit per each sub-module.

Reply:
sure, I kept it separated so we can decide how to organize the code, but I can move it there

Reply:
done (commit 51bd73c)